packages/lundium/package.json

```diff
@@ -22,29 +22,29 @@
 	"dependencies": {
 		"invariant": "^2.2.4",
 		"ramda": "^0.27.0",
-		"ramda-extension": "^0.10.1",
-		"react-helmet": "^5.2.1",
-		"react-popper-tooltip": "^2.10.1",
+		"ramda-extension": "^0.11.0",
+		"react-helmet": "^6.1.0",
+		"react-popper-tooltip": "^4.3.0",
 		"react-transition-group": "^4.3.0"
 	},
 	"devDependencies": {
 		"@babel/cli": "^7.8.4",
-		"@rollup/plugin-replace": "^2.3.1",
+		"@rollup/plugin-replace": "^3.0.0",
 		"enzyme": "^3.11.0",
-		"fs-extra": "^8.1.0",
+		"fs-extra": "^10.0.0",
 		"glob": "^7.1.6",
 		"path": "^0.12.7",
-		"rollup": "^1.32.0",
+		"rollup": "^2.56.3",
 		"rollup-plugin-babel": "^4.3.3",
 		"rollup-plugin-commonjs": "^10.1.0",
 		"rollup-plugin-node-globals": "^1.4.0",
 		"rollup-plugin-node-resolve": "^5.2.0",
-		"rollup-plugin-size-snapshot": "^0.11.0",
-		"rollup-plugin-terser": "^5.2.0"
+		"rollup-plugin-size-snapshot": "^0.12.0",
+		"rollup-plugin-terser": "^7.0.2"
 	},
 	"peerDependencies": {
 		"prop-types": "^15.7.2",
-		"react": "^16.12.0",
-		"react-dom": "^16.12.0"
+		"react": "^17.0.2",
```

Comment:
so you say that Lundium will not work in react 16? I think that is not true.

Reply:
fixed